doc/en/api/1.0.0/templates.yaml

```diff
@@ -114,6 +209,289 @@ paths:
           description: OK
           schema:
             $ref: "#/definitions/Templates"
+    post:
+      operationId: templatesCoveragePost
+      description: Invalid. PUT to /{template} to edit a template.
+      responses:
+        405:
+          description: Method Not Allowed
+    put:
+      operationId: templatesCoveragePut
+      description: Invalid. PUT to /{template} to edit a template.
+      responses:
+        405:
+          description: Method Not Allowed
+    delete:
+      operationId: templatesCoverageDelete
+      description: Invalid. Delete from /{template} to remove a template.
```

Comment:
Is the description markdown? Do you need to use:

```
Invalid. Delete from `/{template}` to remove a template.
```

Reply:
That would probably be better, yes.